extensions.md

```diff
@@ -186,3 +186,53 @@ the server, but LSP does not expose them.
   - Clients that don't support client-side filtering should ignore this and
     use `sortText`, which incorporates `score` and fuzzy-matching.
 
+## AST
+{:v12}
+
+C++ has a complicated grammar and semantic structure that's not always obvious
+from the source. Inspecting the clang AST can lend some insight.
+Despite the name, it captures semantics as well as syntax (e.g. implicit casts).
```

Comment:
Maybe expand AST somewhere because if you're arguing about the name it might look awkward to some people who don't know what AST is in the first place :)